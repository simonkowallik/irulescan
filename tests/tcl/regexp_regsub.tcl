regexp -- exp string ?matchVar? ?subMatchVar subMatchVar ...?
regsub -- exp string subSpec ?varName?

regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline -start <index> -- exp string ?matchVar? ?subMatchVar subMatchVar ...?
regsub -all -expanded -line -linestop -lineanchor -nocase -start <index> -- exp string subSpec ?varName?

regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline -start <index> exp_fail_100 string ?matchVar? ?subMatchVar subMatchVar ...?
regsub -all -expanded -line -linestop -lineanchor -nocase -start <index> exp_fail_200 string subSpec ?varName?

regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline exp_fail_300 string ?matchVar? ?subMatchVar subMatchVar ...?
regsub -all -expanded -line -linestop -lineanchor -nocase exp_fail_400 string subSpec ?varName?

regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline exp_fail_500 string ?matchVar? ?subMatchVar subMatchVar ...?
regsub -all -expanded -line -linestop -lineanchor -nocase exp_fail_600 string subSpec ?varName?

regexp exp_fail_700 string ?matchVar? ?subMatchVar subMatchVar ...?
regsub exp_fail_800 string subSpec ?varName?
