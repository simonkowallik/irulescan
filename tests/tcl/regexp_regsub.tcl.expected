DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_100` in `regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline -start <index> exp_fail_100 string ?matchVar? ?subMatchVar subMatchVar ...?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_200` in `regsub -all -expanded -line -linestop -lineanchor -nocase -start <index> exp_fail_200 string subSpec ?varName?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_300` in `regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline exp_fail_300 string ?matchVar? ?subMatchVar subMatchVar ...?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_400` in `regsub -all -expanded -line -linestop -lineanchor -nocase exp_fail_400 string subSpec ?varName?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_500` in `regexp -about -expanded -indices -line -linestop -lineanchor -nocase -all -inline exp_fail_500 string ?matchVar? ?subMatchVar subMatchVar ...?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_600` in `regsub -all -expanded -line -linestop -lineanchor -nocase exp_fail_600 string subSpec ?varName?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_700` in `regexp exp_fail_700 string ?matchVar? ?subMatchVar subMatchVar ...?`
DANGEROUS: missing options terminator `--` permits argument injection at `exp_fail_800` in `regsub exp_fail_800 string subSpec ?varName?`

